### ISSUE TITLE
Rich colours

### ISSUE THREAD

**Issue by Wildhoney** [Hkattelu/Asciify#4]
Fairly new to Rust so apologies if I'm missing something obvious. I came across your lib from Reddit 👍 

BUT I only get the basic colours when using the lib. Your bird example looks more like Elmer the Patchwork Elephant.

What am I doing wrong? 😢

<img width="1088" alt="Screenshot 2022-09-08 at 09 53 52" src="https://user-images.githubusercontent.com/1528477/189080099-53eabac6-f33e-42c0-b424-73f8c08598b8.png">

```rust
pub fn main() {
    let path = PathBuf::from("./src/bird.png");
    AsciiBuilder::new_from_path(path).set_deep(true).set_resize((40, 20)).to_std_out(true);
}
```

**Comment by Hkattelu** [comment]
The lib you probably found on reddit was [this one](https://github.com/edelsonc/asciify) - I haven't posted on reddit about this lib before. That being said, thanks for checking it out and filing an issue! Second, it is a chocobo, not a bird. Third, I have verified that the program still works fine on my end. The characters themselves look exactly right. The color simply gets copied from the pixels of the image.

Two things stand out in your issue - your terminal colors, and the image itself. First, can you confirm that the image `bird.png`  wasn't modified in any way from the test image? If the file is encoded with some start characters, I could see that screwing up the colors. 

Another possible issue is that the terminal  you are using [doesn't support ANSI colors](https://docs.rs/termcolor/latest/termcolor/trait.WriteColor.html#required-methods). Can you try running the same command with a different terminal?

**Comment by Wildhoney** [comment]
Yep I haven't modified it, I just renamed the image. Will of course rename to `chocobo.png` with my new knowledge 😆 

I've not had an issue with ANSI colours in my terminal before, for example I have `nyan` setup which comes through perfectly, so it's difficult to pinpoint the actual issue.

I'm using zsh currently &ndash; what do you use?

<img width="1442" alt="Screenshot 2022-09-09 at 18 54 40" src="https://user-images.githubusercontent.com/1528477/189414354-492adb54-8500-445e-874d-e48f1f13a5f1.png">
